app/controllers/amp_articles_controller.rb

```diff
@@ -0,0 +1,22 @@
+class AmpArticlesController < ActionController::Base
+  include NotFound
```

Comment:
Should I copy this line: https://github.com/moneyadviceservice/frontend/blob/master/app/controllers/application_controller.rb#L2